kolibri/plugins/coach/assets/src/modules/classSummary/index.js

```diff
@@ -225,6 +226,14 @@ export default {
         Object.values(state.examLearnerStatusMap).map(learnerMap => Object.values(learnerMap))
       );
     },
+    /*
+     * quizzesSizes := [
+     *   { quiz_id: size in bytes for humans }, ...
+     * ]
+     */
+    quizzesSizes(state) {
```

Comment:
I am guessing this is the incomplete vuex state? As it appears to be identical to the getter for exams?